IDE/ROWLEY-CROSSWORKS-ARM/kinetis_hw.c

```diff
@@ -130,6 +143,13 @@ static void hw_gpio_init(void)
         | SIM_SCGC5_PORTE_MASK
 #endif
    );
+
+#if 1 /* Debug clock */
```

Comment:
I would leave this disabled by default `#if 0`. This pin only exists like this for the K64.